client/state/products-list/selectors.js

```diff
@@ -0,0 +1,3 @@
+export function isFetching( state ) {
```

Comment:
In case we ever migrate to a global selectors directory, or at least have a single module from which to import selectors (e.g. `import { isFetching } from 'state/selectors';`) we should name selectors to avoid naming conflicts.

Related: #5954